pkg/store/instance.go

```diff
@@ -164,3 +168,32 @@ func ReadPIDFile(path string) (int, error) {
 	}
 	return pid, nil
 }
+
+type FormatData struct {
+	Instance
+	HostOS       string
+	HostArch     string
+	LimaHome     string
+	IdentityFile string
+}
+
+func AddGlobalFields(inst *Instance) (FormatData, error) {
+	var data FormatData
+	data.Instance = *inst
+	// Add HostOS
+	data.HostOS = runtime.GOOS
+	// Add HostArch
+	data.HostArch = runtime.GOARCH
```

Comment:
I think we need to translate the `GOARCH` values (`amd64`, `arm64`) here to the `Arch` values we use everywhere else (`x86_64`, `aarch64`).

Reply:
Should it use limayaml, or move the constants somewhere else ?

```
pkg/qemu/qemu.go:	nativeX8664 := arch == limayaml.X8664 && runtime.GOARCH == "amd64"
pkg/qemu/qemu.go:	nativeAARCH64 := arch == limayaml.AARCH64 && runtime.GOARCH == "arm64"
```

Reply:
Using the `limayaml` constants sounds fine to me. After all, that's what we want to be consistent with. 😄

Reply:
Ok, just need to export the translator and it should be done